.github/workflows/ci.yaml

```diff
@@ -134,6 +129,24 @@ jobs:
           npm install -g license-checker
           license-checker --production --json --onlyAllow="${ALLOWED_LICENSES}" --excludePackages="${EXCLUDE_PACKAGES}"
 
+  build-docker:
```

Comment:
Remember to add the  `needs` part with at the very least `is-safe-to-run`. Without it, the sensitive actions will run even if the PRs aren't labeled safe to run, which is a security issue.

This same review applies to every repo with a `pull_request_target` trigger